src/components/Slider/constants.ts

```diff
@@ -0,0 +1,30 @@
+const DEFAULT_STEP = 1
+const INIT_POINT = 0
+const MINIMUM_TRACK_WIDTH = 0
+const DEFAULT_MINIMUM_VALUE = 0
+const DEFAULT_MAXIMUM_VALUE = 1
+const INIT_VALUE = 0
+const VISIBLE = 1
+const INVISIBLE = 0
+const NEXT_STEP = 1
+const PREVIOUS_STEP = 1
+const FIRST_POINT = 1
+const DURATION = 200
+
+const ThumbPosition = {left: 'left', right: 'right'}
```

Comment:
Please change `ThumbPosition` to `THUMB_POSITION` and check the relevant files with this const